src/options/options.controllers.ts

```diff
@@ -587,31 +685,38 @@ export async function getAccountFeesBase(req: any): Promise<any> {
       req.connector
     );
   if (connector instanceof Panoptic) {
-    return panopticGetAccountFeesBase(<Ethereumish>chain, connector, req);
+    return panopticGetAccountPremium(<Ethereumish>chain, connector, req);
+  } else {
+    return new Error(`Method undefined on this connector, or no valid connector.`);
   }
 }
 
-export async function calculateDelta(req: any): Promise<any> {
+export async function getAccountFeesBase(req: GetAccountFeesBaseRequest): Promise<GetAccountFeesBaseResponse | Error> {
+  const chain = await getInitializedChain<Ethereumish>(req.chain, req.network);
   const connector: Panoptic =
     await getConnector<Panoptic>(
       req.chain,
       req.network,
       req.connector
     );
   if (connector instanceof Panoptic) {
-    return panopticCalculateDelta(connector, req);
+    return panopticGetAccountFeesBase(<Ethereumish>chain, connector, req);
+  } else {
+    return new Error(`Method undefined on this connector, or no valid connector.`);
   }
 }
 
-export async function calculateGamma(req: any): Promise<any> {
+export async function addLeg(req: CreateAddLegsRequest): Promise<CreatePositionResponse | Error> {
+  const chain = await getInitializedChain<Ethereumish>(req.chain, req.network);
   const connector: Panoptic =
     await getConnector<Panoptic>(
       req.chain,
       req.network,
       req.connector
     );
   if (connector instanceof Panoptic) {
-    return panopticCalculateGamma(connector, req);
+    return panopticAddLeg(<Ethereumish>chain, connector, req);
+  } else {
+    return new Error(`Method undefined on this connector, or no valid connector.`);
   }
-}
-
+}
```

Comment:
Nit: could add a newline to the end of this file

Reply:
Done: a2fbcbb63a7929c8c84b979c1ec6fb9514657675